src/app/game-state/home.service.ts

```diff
@@ -474,7 +474,8 @@ export class HomeService {
         this.ageFields();
         if (this.home.costPerDay > this.characterService.characterState.money){
           this.logService.addLogMessage("You can't afford the upkeep on your home. Some thugs rough you up over the debt. You better get some money, fast.", "INJURY", 'EVENT');
-          this.characterService.characterState.status.health.value -= 20;
+          //Thugs want money and they can't collect if you're dead, so scare the player. But the rats and monsters want you dead, and this makes for easier pickings and it scales with health.
+          this.characterService.characterState.status.health.value *= 0.5;
```

Comment:
Let's leave this change out. In the early game I want the feel of constant danger, and this would remove a lot of that. It would also be very different from what I want in the late game. When you have a lot of health and health recovery, you can mostly shrug off the thugs, allowing strategies like growing your spirituality through fasting.

Reply:
Removed.